content/download.html

```diff
@@ -106,52 +101,58 @@ <h3>Download OpenRA for OS X</h3>
   <pre>$ brew cask install openra</pre>
   <p class="downloadthirdparty">We do not maintain this external package source, so there may be delays when a new version is released.<br />Please contact the downstream repository maintainers about any packaging issues.</p>
 </div>
-<div id="deb" class="tab">
+<div id="linux" class="tab">
   <hr />
-  <h3>Download OpenRA for Debian / Ubuntu</h3>
-  <p>OpenRA requires Mono 4.2 or greater. Ubuntu 14.04 and Debian Jessie users should use the official upstream <a href="https://www.mono-project.com/download/stable/#download-lin">Mono package repository</a> to upgrade their Mono installation.</p>
-  <p>Download and run one of the packages below to install OpenRA via the Software Center.</p>
+  <h3><img src="/images/platforms/appimage-small.svg" class="downloadsmallplatform" />Official AppImages</h3>
+  <p>OpenRA is distributed as portable <a href="https://appimage.org/">AppImages</a> that run on most modern 64-bit Linux distributions.<br />Mono 4.2 or later is required. <a href="https://www.mono-project.com/download/stable/#download-lin">Official Mono packages</a> can be installed if your distro packages are not compatible.</p>
+  <p>Download the mods you want to play, <a href="https://github.com/OpenRA/OpenRA/wiki/AppImages">make the AppImage files executable</a>, then double click to play!<br />Install the optional <a href="https://github.com/AppImage/appimaged">appimaged</a> utility to integrate OpenRA with your system launcher and web browser.</p>
+  <br />
```

Comment:
If you want to do this with a `<br>`, consider to use one after the following closing `</ul>` too.`